programs/models.py

```diff
@@ -162,15 +162,24 @@ def __str__(self):
 
 
 class DocumentManager(models.Manager):
-    translated_fields = ("text",)
+    translated_fields = ("text", "link_url", "link_text")
+    no_auto_fields = "link_url"
```

Comment:
This needs to be a list.
```suggestion
    no_auto_fields = ("link_url",)
```

Reply:
I am just going to commit this because it is one line, so we can release it today.